docs/model/assertions/overview.rst

```diff
@@ -1,11 +1,12 @@
 Assertions Overview
 ===================
+An assertion classifies the clinical significance of a variant-disease association under recognized guidelines (see AMP Classification, NCCN Guidelines, for more).
 
 .. image:: /images/figures/CIViC_assertion-fields_v1j.png
 
-Lorem ipsum dolor sit amet, consectetuer adipiscing elit.  Donec hendrerit tempor tellus.  Donec pretium posuere tellus.  Proin quam nisl, tincidunt et, mattis eget, convallis nec, purus.  Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.  Nulla posuere.  Donec vitae dolor.  Nullam tristique diam non turpis.  Cras placerat accumsan nulla.  Nullam rutrum.  Nam vestibulum accumsan nisl.
+The CIViC Assertion (AID) summarizes a collection of Evidence Items (EIDs) that covers predictive/therapeutic, diagnostic, prognostic or predisposing clinical information for a variant in a specific cancer context (Figure 4). Functional Assertions are not currently supported. In general, an Assertion of a certain Type (e.g. Diagnostic) will be supported by a collection of Evidence Items (EIDs) of the same type, but Functional EIDs may be used to support Assertions of other types. The collection of EIDs associated with an Assertion should cover the important clinically relevant findings for the variant in the context of the specific cancer (and drug entity for Predictive/Therapeutic Assertions). Assertion summaries mention NCCN or other practice guidelines related to the variant, as well as FDA drug approvals. In place of the curator assigned star rating found in CIViC Evidence Items, CIViC Assertions utilize widely adopted, published guidelines for variant tiering and pathogenicity assessment, which are detailed below. CIViC Assertions are the primary entity that make up CIViC submissions to ClinVar.
 
-Aliquam erat volutpat.  Nunc eleifend leo vitae magna.  In id erat non orci commodo lobortis.  Proin neque massa, cursus ut, gravida ut, lobortis eget, lacus.  Sed diam.  Praesent fermentum tempor tellus.  Nullam tempus.  Mauris ac felis vel velit tristique imperdiet.  Donec at pede.  Etiam vel neque nec dui dignissim bibendum.  Vivamus id enim.  Phasellus neque orci, porta a, aliquet quis, semper a, massa.  Phasellus purus.  Pellentesque tristique imperdiet tortor.  Nam euismod tellus id erat.
+CIViC currently has two main types of Assertions: those based on variants of primarily somatic origin (predictive/therapeutic, predisposing, and diagnostic) and those based on variants of primarily germline origin (predisposing). When the number and quality of predictive/therapeutic, prognostic or diagnostic Evidence Items (EIDs) in CIViC sufficiently cover what is known for a particular variant and cancer type, then a corresponding assertion be created in CIViC.
```

Comment:
I think there might be a mistake here with the listed assertion types for somatic vs germline. Predisposing is listed twice and prognostic is not listed at all. The second sentence lists prognostic but not predisposing.